css/style.css

```diff
@@ -27,7 +27,7 @@ footer {
   color: white;
   padding
 }
-.float {
+.right-align {
```

Comment:
Sorry, what I meant was: you should make clear that this class will move the element on the right side..

e.g: 'float-right', 'align-right', ecc... something like that

Reply:
so i changed it to .right-align, the same thing as .align-right